src/modules/scraper/adapter/messaging/FindMissedFillEventConsumer.ts

```diff
@@ -78,6 +78,7 @@ export class FindMissedFillEventConsumer {
           { id: jobId },
           { status: FindMissedFillEventJobStatus.Suspended },
         );
+        return;
```

Comment:
is this necessary

Reply:
Ah I see that we need it to fix that bug

Reply:
Yep. That's because we retry fail messages every minute. But for suspended jobs we don't want to retry.
So this suspended jobs are being retried over and over again.